sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -596,6 +600,10 @@ def test_tan_rewrite():
     assert tan(x).rewrite(sec) == sec(x)/sec(x - pi/2, evaluate=False)
     assert tan(x).rewrite(csc) == csc(-x + pi/2, evaluate=False)/csc(x)
     assert tan(sin(x)).rewrite(Pow) == tan(sin(x))
+    assert tan(pi*Rational(2, 5), evaluate=False).rewrite(sqrt) == sqrt(sqrt(5)/8 +
+               Rational(5, 8))/(Rational(-1, 4) + sqrt(5)/4)
```

Comment:
When breaking a long line try to make it easy to read what is happening by choosing a goof place to break the line e.g. by separating the operation from the answer:
```
    assert tan(pi*Rational(2, 5), evaluate=False).rewrite(sqrt) ==\
        sqrt(sqrt(5)/8 + Rational(5, 8))/(Rational(-1, 4) + sqrt(5)/4)
```